zeppelin-server/pom.xml

```diff
@@ -109,6 +110,12 @@
       </exclusions>
     </dependency>
 
+        <dependency>
+          <groupId>org.apache.directory.server</groupId>
+          <artifactId>kerberos-client</artifactId>
+          <version>${kerberos-client.version}</version>
```

Comment:
Why we need kerberos-client in zeppelin-server ?

Reply:
When i tried to build with hadoop3 profile, the build was failed and it throws an error `[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project zeppelin-server: Compilation failure: Compilation failure:
[ERROR] /home/bhavik/workspace/apache_zep/zeppelin/zeppelin-server/src/main/java/org/apache/zeppelin/realm/kerberos/KerberosUtil.java:[41,58] package org.apache.directory.server.kerberos.shared.keytab does not exist`

Reply:
Then I believe you need to put it under profile hadoop3